src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
That all sounds fair to me and I'd agree that I don't see a semantic difference between `interface=none` and a non-existent interface. But in that line, I would question why a `none` type even needs to be mentioned in the Spec at all, as it would essentially be taking the place of my (possibly misguided) introduction of the dummy `undefined` type.